docker/bungeecord/Dockerfile

```
@@ -1,9 +1,13 @@
 FROM maven:3.6.3-jdk-8 as redisbungee-provider
 WORKDIR /
+RUN apt update
+RUN apt install patch -y
```

```suggestion
RUN apt-get update && apt-get install patch -y && apt-get clean && rm -rf /var/lib/apt/lists/*
```
See: https://docs.docker.jp/engine/articles/dockerfile_best-practice.html

---

いらないですね、はい

---

スクリプト中では`apt`より`apt-get`を利用したほうが望ましいのですね。ありがとうございます。